src/dbus/abrt_problems2_entry.c

```diff
@@ -966,10 +966,16 @@ uid_t abrt_p2_entry_get_owner(AbrtP2Entry *entry,
     {
         g_set_error(error, G_DBUS_ERROR, G_DBUS_ERROR_IO_ERROR,
                     "Failed open dump directory");
-        return -1;
+        return 0;
```

Comment:
This would be indistinguishable from root onwership, no? Maybe `abrt_p2_service_notify_entry_object()` should handle failures a bit better?

Reply:
Oh, wait, it’s below in the diff.

Reply:
Still, I think -1 would stand out better.